README.md

```diff
@@ -1,18 +1,31 @@
-Sparkbox Design Systems Survey Microsite 2019
-======================
+Sparkbox Design Systems Survey
+==============================
+
+## Setup
+
+The build system for this website uses Node and Gatsby, with each survey being a microsite subpage. The following steps explain how to setup your local environment after pulling down the repository:
 
-Setup
------
 1. Before running the project setup node/npm ([Installation instructions](https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager)).
 
 2. Install the ([Gatsby CLI](https://www.gatsbyjs.org/docs/quick-start#install-the-gatsby-cli)) globally `npm install -g gatsby-cli`
 
 3. Run `npm install` to install all packages and dependencies. (If you have a M1 Mac, you may need to run `brew reinstall vips` before installing the site packages and dependencies.)
 
-4. Run `npm start`. This will start a hot-reloading development environment at `localhost:8000`.
+## Running Latest Microsite
+
+1. Run `npm start` in your shell terminal
+2. A hot-reloading development environment of the latest version will start at `http://localhost:8000`.
+3. Open [localhost:8000/2022](http://localhost:8000/2022) in your browser to view the site
+
+## Run all years (except 2018)
+1. In your shell run `npm run start:all`
```

Comment:
There is no start:all script.  Is this one of the issues that is fixed in a future PR?

Reply:
Oh, not really. That was my first attempt that I removed from this branch, but didn’t realized I left that reference in the README. Removed, thank you.